testsupport/init.go

```diff
@@ -66,6 +66,11 @@ func WaitForDeployments(t *testing.T, obj runtime.Object) (*framework.Context, *
 	}
 	hostAwait.RegistrationServiceURL = registrationServiceURL
 
+	// setup host metrics route for metrics verification in tests
+	hostMetricsRoute, err := hostAwait.SetupRouteForService("host-operator-metrics", "/metrics")
```

Comment:
Since we create the route here, then I guess that we can remove it from here: https://github.com/codeready-toolchain/toolchain-e2e/blob/300d9be715cdacb6df2811834cda92a64f7b5886/test/perf/perf_test.go#L44

Reply:
Good catch! Thanks! Done in d1ffef3bfd12c67c8135e89acaeedfca3cc0206a